docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
```

Comment:
Non-blocking: I'd prefer if we used an earlier syntax, since then users who (somehow) have older versions of docker don't need to update to run this.